src/main/java/StringCalc/StringAddCalculator.java

```diff
@@ -0,0 +1,59 @@
+package StringCalc;
+
+import java.util.Objects;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringAddCalculator {
+    public static int splitAndSum(String inputText) {
+        if (!validText(inputText)) {
+            return 0;
+        }
+        String[] operandArr = getOperandArr(inputText);
+        if (operandArr.length == 1) {
+            return 1;
+        }
+
+        return sum(operandArr);
+    }
+
+    public static String[] getOperandArr(String text) {
+        Matcher m = Pattern.compile("//(.)\n(.*)").matcher(text);
+        String customDelimiter;
+        String regExp = "[,:^]+";
+        if (m.find()) {
+            customDelimiter = m.group(1);
+            text = m.group(2);
+            regExp = "[,:"+customDelimiter+"^]+";
+        }
+        return text.split(regExp);
+    }
+
+    public static int validNumber(String number) {
+        int parseNumber = 0;
+        try {
+            parseNumber = Integer.parseInt(number);
+        } catch (NumberFormatException e) {
+            throw new RuntimeException();
+        }
+        if (parseNumber < 0) {
+            throw new RuntimeException();
+        }
+        return parseNumber;
+    }
+
+    public static Boolean validText(String inputText) {
+        if (Objects.isNull(inputText) || inputText.trim().length() == 0) {
+            return false;
+        }
+        return true;
```

Comment:
```suggestion
               return !Objects.isNull(inputText) && inputText.trim().length() != 0;
```
취향차이일 순 있는데 저는 한줄로 쓰는것을 선호합니다. 혜우님은 어떠신가요~?